dle/dle/settings.py

```diff
@@ -41,18 +41,19 @@
 STATIC_URL = "/static/"
 
 NLP_MODELS = os.path.join(BASE_DIR, "api/bert_models")
-
+URL = os.environ.get("SEARCHRX_URL")
 # Hosts and CIDR (AWS subnets)
 try:
     ALLOWED_HOSTS = [
+        URL,
         "druglabelexplorer.org",
         "www.druglabelexplorer.org",
         "127.0.0.1",
         "localhost",
         "testserver",
     ] + env.list("ALLOWED_HOSTS")
```

Comment:
This line means you should just be able to set `ALLOWED_HOSTS=searchrx-1695064113.us-east-1.elb.amazonaws.com,http://anotherurl.com,etc`. No need to modify this. We should actually take out the old `druglabelexplorer` URLs.

Reply:
But, not sure I agree hard-coding the load balancer is the best way to go. By taking a parameterized approach we will able to modify as needed.
ya we should probably remove the old ones.

Reply:
However, not sure if removing the old ones is relevant to this commit.

Reply:
Update: they actually already a parameterized it with ALLOWED_HOSTS var. We just were'nt using it before .

I'll update and re-submit